DOL.WHD.Section14c.Web/.eslintrc.js

```diff
@@ -49,7 +49,7 @@ module.exports = {
         //Best Practices
         "accessor-pairs": 2,
         "block-scoped-var": 0,
-        "complexity": [2, 6],
+        "complexity": [2, 12],
```

Comment:
Please justify changing the lint complexity rule (especially doubling it) over changing the code.  A cyclomatic complexity over 5-6 deserves attention and over 10 deserves serious thought.  In some rare cases it can't be helped - if that's the case here, consider disabling the lint for the specific method with the high complexity rather than changing the global rule.